OrbitSshQt/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ target_link_libraries(OrbitSshQtIntegrationTests PUBLIC OrbitSshQt GTest::GTest)
 
 # tests
```

Comment:
What happened with this commented code?

Reply:
hmm it was commented out in 68cbe813f018d56efce3fd96dff42a9c9e80db11:

commit 68cbe813f018d56efce3fd96dff42a9c9e80db11
Author: antonrohr <antonrohr@google.com>
Date:   Tue Jun 2 09:22:42 2020 +0200

    Remove unused SshQtTests.
    
    There was a typo in OrbitSshQt/CMakeLists.txt that ended in the
    OrbitSshTest being registered twice. This is fixed now

@antonrohr should this really be disabled?